BSB_LAN/BSB_LAN.ino

```diff
@@ -2956,8 +2964,13 @@ void LogTelegram(byte* msg) {
     default: logThis = false; break;
   }
   if (logThis) {
-    dataFile = SDCard.open(journalFileName, FILE_APPEND);
-    if (dataFile) {
+    if (LoggingMode & CF_LOGMODE_MQTT) {
+      printlnToDebug(PSTR("Send to MQTT"));
+      if ((decodedTelegram.prognr != -1) and (!is_query)) {
+        mqtt_sendtoBroker(-1);
```

Comment:
How do you know that the destination address is always the default destination address (that's what -1 stands for in `param.dest_addr`)?